support/ruby/consolr/lib/consolr.rb

```diff
@@ -142,36 +145,27 @@ def start
         puts "Cannot run dangerous commands on #{@node.hostname} (#{@node.tag} - #{@node.status})"
         abort dangerous_body
       end
-      
+
       case
       when options[:console]
-        puts "--> Opening SOL session (type ~~. to quit)"
-        puts self.ipmitool_cmd("sol activate")
-      
+        puts '--> Opening SOL session (type ~~. to quit)'
+        puts ipmitool_cmd('sol activate')
```

Comment:
You might like:
puts '--> Opening SOL session (type ~~. to quit)', ipmitool_cmd('sol activate')
or even:
puts '--> Opening SOL session (type ~~. to quit)',
   ipmitool_cmd('sol activate')